src/components/Issue/Issue.tsx

```diff
@@ -7,49 +7,69 @@ import {
 } from "@material-ui/core";
 import React, { Component, ReactNode } from "react";
 import { Status } from "common/constants";
-import { IIssue } from "common/models";
 import 'components/Issue/Issue.css';
+import { IIssue } from "common/models";
+import EditIssue from "components/Issue/CreateEditIssue/EditIssue/EditIssue";
+
+interface IssueProps {
+  issue: IIssue;
+}
 
-interface IIssueProps {
-  issue: IIssue,
-  id: number,
+interface IssueState {
+  showEditButton: boolean;
 }
 
-class Issue extends Component<IIssueProps> {
+class Issue extends Component<IssueProps, IssueState> {
+  constructor(props: IssueProps) {
+    super(props);
+
+    this.state = {
+      showEditButton: false
+    }
+  }
+
   render(): ReactNode {
+    const { title, description, status, assignee } = this.props.issue;
+
     let statusColor = "green";
 
-    if (this.props.issue.status === Status.InProgress) {
+    if (status === Status.InProgress) {
```

Comment:
You can make switch statement here which defaults to green color. It's just more raad-able, no big deal at ll

Reply:
done